src/Microsoft.AspNetCore.Razor.Language/ClassifiedSpanVisitor.cs

```diff
@@ -161,15 +161,21 @@ public override SyntaxNode VisitHtmlLiteralAttributeValue(HtmlLiteralAttributeVa
             return base.VisitHtmlLiteralAttributeValue(node);
         }
 
-        public override SyntaxNode VisitHtmlTextLiteral(HtmlTextLiteralSyntax node)
+        public override SyntaxNode VisitMarkupTextLiteral(MarkupTextLiteralSyntax node)
         {
             if (node.Parent is HtmlLiteralAttributeValueSyntax)
             {
-                return base.VisitHtmlTextLiteral(node);
+                return base.VisitMarkupTextLiteral(node);
             }
 
             WriteSpan(node, SpanKindInternal.Markup);
-            return base.VisitHtmlTextLiteral(node);
+            return base.VisitMarkupTextLiteral(node);
+        }
+
+        public override SyntaxNode VisitMarkupEscapedTextLiteral(MarkupEscapedTextLiteralSyntax node)
```

Comment:
Wait,`MarkupEscapedTextLiteralSyntax` and the CSharp equivalent ones are only used for escaped transitions right? is `EscapedTextLiteral` the right wording?

Reply:
Spoke offline. This is for more cases than just the transitions. Going to rename this to `MarkupEphemeralTextLiteralSyntax` to indicate these are represented in the tree but are not significant and won't be rendered in the output (This is one of the scenarios that should be considered to represent as trivia)